app/src/main/java/com/example/lunchtray/LunchTrayScreen.kt

```diff
@@ -134,8 +136,9 @@ fun LunchTrayApp() {
                         viewModel.updateEntree(item)
                     },
                     modifier = Modifier
-                        .padding(dimensionResource(R.dimen.padding_medium))
                         .verticalScroll(rememberScrollState())
+                        .padding(innerPadding)
```

Comment:
Combine the two `.padding()`  function calls into 1 or remove 1 depending on what you need.